prow/gcpproxy-api-regression.sh

```diff
@@ -58,4 +57,14 @@ git checkout ${SHA}
 echo ${VERSION} > ${ROOT}/VERSION
 
 make depend.install
-make integration-test-without-envoy-build
+make build build-envoy build-grpc-interop build-grpc-echo
+
+echo '======================================================='
+echo '============ Download latest configmanager ============'
```

Comment:
Honestly we can remove this download dependency right? Building config manager is really quick, I don't think you're saving much time here. We can just build config manager before checking out `master` on line 48.

Reply:
Done.